### PR TITLE
Add task for EUC + mTLS authentication

### PR DESCRIPTION
**Pull request by diemtvu**

### CODE REVIEW

File: content/docs/tasks/security/authn-policy/index.md [hunk 1]
```diff
@@ -571,16 +571,82 @@ $ for i in `seq 1 10`; do curl --header "Authorization: Bearer $TOKEN" $INGRESS_
...
+Therefore, you do not need adding this destination rule. On the other hand, you still need to add the `mTLS` stanza to the authentication policy as the service-specific policy will override the mesh-wide (or namespace-wide) policy completely.
+
+After these changes, traffic from Istio services, including ingress gateway, to `httpbin.foo` will use mutual TLS. The test command above will still work. Requests from Istio-services directly to `httpbin.foo` also work, given the correct token:
+
+{{< text bash >}}
+$ kubectl exec $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -n foo -- curl http://httpbin.foo:8000/ip -s -o /dev/null -w "%{http_code}\n" --header "Authorization: Bearer $TOKEN"
+200
 {{< /text >}}
+
+However, requests from non-Istio service, which use plain-text will fail:
```
Comment (geeknoid) [inline content/docs/tasks/security/authn-policy/index.md]: service -> services

File: content/docs/tasks/security/authn-policy/index.md [hunk 2]
```diff
@@ -571,16 +571,82 @@ $ for i in `seq 1 10`; do curl --header "Authorization: Bearer $TOKEN" $INGRESS_
...
+  trafficPolicy:
+    tls:
+      mode: ISTIO_MUTUAL
+EOF
+{{< /text >}}
+
+> If you already enable mutual TLS mesh-wide or namespace-wide, the host `httpbin.foo` is already covered by the other destination rule.
+Therefore, you do not need adding this destination rule. On the other hand, you still need to add the `mTLS` stanza to the authentication policy as the service-specific policy will override the mesh-wide (or namespace-wide) policy completely.
+
+After these changes, traffic from Istio services, including ingress gateway, to `httpbin.foo` will use mutual TLS. The test command above will still work. Requests from Istio-services directly to `httpbin.foo` also work, given the correct token:
```
Comment (geeknoid) [inline content/docs/tasks/security/authn-policy/index.md]: Istio-services -> Istio services

File: content/docs/tasks/security/authn-policy/index.md [hunk 3]
```diff
@@ -571,16 +571,82 @@ $ for i in `seq 1 10`; do curl --header "Authorization: Bearer $TOKEN" $INGRESS_
...
+    {{< text bash >}}
+    $ kubectl delete policy jwt-example
+    {{< /text >}}
+
+1. Remove destination rule:
+
+    {{< text bash >}}
+    $ kubectl delete policy httpbin
+    {{< /text >}}
+
```
Comment (geeknoid) [inline content/docs/tasks/security/authn-policy/index.md]: Remove extra blank line for linting to pass.

File: content/docs/tasks/security/authn-policy/index.md (hunk 1 above)
Comment (diemtvu) [reply content/docs/tasks/security/authn-policy/index.md]: Done

File: content/docs/tasks/security/authn-policy/index.md (hunk 3 above)
Comment (diemtvu) [reply content/docs/tasks/security/authn-policy/index.md]: Done

File: content/docs/tasks/security/authn-policy/index.md (hunk 2 above)
Comment (diemtvu) [reply content/docs/tasks/security/authn-policy/index.md]: Done

**Pull request merged**